octopoes/octopoes/api/router.py

```diff
@@ -568,18 +568,18 @@ def importer(data: bytes, xtdb_session_: XTDBSession, reset: bool = False) -> di
 
 
 @router.post("/io/import/add", tags=["io"])
-async def importer_add(request: Request, xtdb_session_: XTDBSession = Depends(xtdb_session)) -> dict[str, int]:
+def importer_add(request: Request, xtdb_session_: XTDBSession = Depends(xtdb_session)) -> dict[str, int]:
     try:
-        data = await request.body()
+        data = request.body()
     except Exception as e:
         raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR, detail="Error receiving objects") from e
     return importer(data, xtdb_session_)
 
 
 @router.post("/io/import/new", tags=["io"])
-async def importer_new(request: Request, xtdb_session_: XTDBSession = Depends(xtdb_session)) -> dict[str, int]:
+def importer_new(request: Request, xtdb_session_: XTDBSession = Depends(xtdb_session)) -> dict[str, int]:
     try:
-        data = await request.body()
+        data = request.body()
```

Comment:
Here we do call async code though?

Reply:
Yes, so I need to fix it differently. I've added `sync_to_async` decorator to the importer so we can await the importer. Even if you call async code, you can't just mix sync and async code.